install_config/uninstall-olm-using-ansible.adoc

```diff
@@ -0,0 +1,15 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+:_content-type: ASSEMBLY
+[id="uninstall-olm-using-ansible"]
+= Uninstalling Operator Lifecycle Manager using Ansible
```

Comment:
To me, it looks a little off that there are two headings exactly the same back to back

![image](https://user-images.githubusercontent.com/7443887/157459715-5712bc8d-6706-4f70-bc90-42fe4ecedaa6.png)

I would suggest to change them to be not exactly the same. What I'd recommend is to update the topic map and assembly title to be just "Uninstalling Operator Lifecycle Manager", and then leave the below file's title having that "using Ansible" part

Reply:
Addressed